src/wp-includes/blocks.php

```diff
@@ -1039,17 +1039,24 @@ function set_ignored_hooked_blocks_metadata( &$parsed_anchor_block, $relative_po
  *
  * @since 6.6.0
  * @since 6.7.0 Injects the `theme` attribute into Template Part blocks, even if no hooked blocks are registered.
+ * @since 6.8.0 Have the `$context` parameter default to `null`, in which case the current post will be used.
```

Comment:
`get_post` can still return `null`. It’s fine type wise, but maybe the documentation should state that it will try to get the current post.

Reply:
I've covered this [here](https://github.com/WordPress/wordpress-develop/pull/7898/files#diff-8c99af92e4ec0fdb307ddd9b42be1e1ef1efe4a9f31287c23f346244dddd1ce9R1047-R1048). I can change it to something like

> If set to `null`, `get_post()` will be called to use the current post as context. Default: `null`.

WDYT?

Additionally, I can change the `@since` line to

```suggestion
 * @since 6.8.0 Have the `$context` parameter default to `null`, in which case `get_post()` will be called to use the current post as context.
```

Reply:
Works for me 👍🏻

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/7898/commits/2a1f4354e5e03cb2faa722cc66d903197e04cc93 and https://github.com/WordPress/wordpress-develop/pull/7898/commits/ff37889f604772494f8d4aeb6cb9c64a1f47606e.